src/napari_imagej/widgets/napari_imagej.py

```diff
@@ -188,3 +212,24 @@ def _finalize_info_bar(self):
         self.widget.info_box.version_bar.setText(
             " ".join(["ImageJ", str(ij().getVersion())])
         )
+
+    def _finalize_exception_printer(self):
+        @JImplements(["org.scijava.event.EventSubscriber"], deferred=True)
+        class NapariEventSubscriber(object):
+            @JOverride
+            def onEvent(self, event):
+                log_debug(str(event))
+
+            @JOverride
+            def getEventClass(self):
+                return jc.SciJavaEvent.class_
+
+            @JOverride
+            def equals(self, other):
+                return isinstance(other, NapariEventSubscriber)
+
+        event_bus_field = ij().event().getClass().getDeclaredField("eventBus")
```

Comment:
Maybe add a HACK linking to https://github.com/scijava/scijava-common/issues/452